MH-FiorellaTheme5/source/stylesheets/site.css.scss

```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: #fbc547;
```

Comment:
Color literals like `#fbc547` should only be used in variable declarations; they should be referred to via variable everywhere else.